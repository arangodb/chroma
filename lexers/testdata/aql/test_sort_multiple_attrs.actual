FOR c IN Characters
    FILTER c.surname
    SORT c.surname, c.name
    LIMIT 10
    RETURN {
        surname: c.surname,
        name: c.name
    }