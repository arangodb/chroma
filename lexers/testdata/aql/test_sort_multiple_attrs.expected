[
  {"type":"KeywordReserved","value":"FOR"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Text","value":" "},
  {"type":"KeywordReserved","value":"IN"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"Characters"},
  {"type":"Text","value":"\n    "},
  {"type":"KeywordReserved","value":"FILTER"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"surname"},
  {"type":"Text","value":"\n    "},
  {"type":"KeywordReserved","value":"SORT"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"surname"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"name"},
  {"type":"Text","value":"\n    "},
  {"type":"KeywordReserved","value":"LIMIT"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberFloat","value":"10"},
  {"type":"Text","value":"\n    "},
  {"type":"KeywordReserved","value":"RETURN"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"Text","value":"\n        "},
  {"type":"Name","value":"surname"},
  {"type":"Operator","value":":"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"surname"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n        "},
  {"type":"Name","value":"name"},
  {"type":"Operator","value":":"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"name"},
  {"type":"Text","value":"\n    "},
  {"type":"Punctuation","value":"}"}
]
